src/viadot/sources/__init__.py

```diff
@@ -26,3 +24,6 @@
 
 if find_spec("pyrfc"):
     __all__.extend(["SAPRFC", "SAPRFCV2"])
+
+if find_spec("databricks-connect"):
+    __all__.extend(["Databricks"])
```

Comment:
if it's a single item you can just use `append()`

Reply:
[🎨 Changed list method in __init__.py](https://github.com/dyvenia/viadot/pull/880/commits/53597e0654b95df4b3e5f5035e42df626546af5a)

Reply:
Yes but you need to append an item, not a list... `a.append("b")`, not `a.append(["b"])`. That's the difference between `append()` and `extend()`.

Reply:
[🐛 Fixed imports in __init__.py](https://github.com/dyvenia/viadot/pull/880/commits/de5124f7cad07eb58db8773de4dad7ca9df8d665)

Reply:
I added imports in the if statement because, in the case of installed, e.g. `pyspark` it was added Databricks to `__all__ `variable but it still didn't imported a Databricks class, which caused an error. So, for example, if we installed Databricks, we couldn't use it.